src/test/java/org/opentripplanner/raptor/moduletests/K01_TransitPriorityTest.java

```diff
@@ -70,10 +70,13 @@ public DominanceFunction dominanceFunction() {
   @BeforeEach
   private void prepareRequest() {
     // Each pattern depart at the same time, but arrive with 60s between them.
-    // Given a slack on the cost equals to ~90s make both L1 and L2 optimal, but no L3
+    // Given a slack on the cost equals to ~90s make both L1 and L2 optimal, but no L3.
+    // Line U1 is not optimal, because it slower than L1 and is in the same group
```

Comment:
I tried to improve:
```suggestion
    // Each pattern depart at the same time, but arrive at different times and they may
    // belong to different groups.
    // Line U1 is not optimal, because it slower than L1 and is in the same group. 
    // Given a slack on the cost equals to ~90s this makes both L1 and L2 optimal (since
    // they are in different groups), but not L3 (which certainly is in its own group but
    // its cost is outside the range allowed by the slack).
```
(See same comment in `K02_TransitPriorityDestinationTest.java`.)